### DISCUSSION TITLE
CSS and JS assets not rendering on LEMP stack with nginx

### DISCUSSION THREAD

**Discussion by Kenneth-Inkum** [filamentphp/filament#5291] [Help]
Filament project won't have any css or js formatted pages in the browser. Inspection indicates that files not found but I can confirm that the files are present in the vendor directory.

![image](https://user-images.githubusercontent.com/54093343/208437147-e23d4940-7fea-4520-a43a-310ff2e36679.png)

How can I resolve this problem. [latest version of Laravel 9 with PHP 8.1.0]

**Comment by danharrin** [comment]
https://github.com/livewire/livewire/issues/242#issuecomment-948054946
